index.bs

```diff
@@ -5679,39 +5674,34 @@ steps may throw an exception.
       <dt>If IsArray(|input|)</dt>
       <dd>
 
-        1. Let |len| be the ToLength(Get(|input|,
+        1. Let |len| be [=!=] ToLength(Get(|input|,
```

Comment:
Ah, yes - an Array exotic's length is an own property, so the prototype's getter is ignored when getting length. Still, I bet proxies do bad stuff, so ? is correct here anyway but it'll be interesting to see if any impl trips up.